create_redirect.yml

```diff
@@ -0,0 +1,83 @@
+---
+# This playbook creates the redirection core app objects.
+
+- hosts: local
+  gather_facts: False
+
+  pre_tasks:
+    - import_tasks: tasks/validate_configuration.yml
+
+  tasks:
+    - name: Display playbook name
+      debug: msg="==== Starting create_redirect playbook ===="
+      tags: deploy
+
+    - import_tasks: tasks/set_vars.yml
+
+    - name: Lookup available core applications
+      set_fact: core_apps="{{ [ lookup('apps', 'core_apps/') ] | flatten }}"
+
+    - name: Set app to point to the redirect app
+      set_fact:
+        app: '{{ core_apps | json_query("[?name==''redirect''] | [0]") }}'
+
+    - name: Set redirect app templates
```

Comment:
from here to line 31, it can be replaced by the task `get_objects_for_app`. It will void repeating code and less code to maintain.

Reply:
True.